roxie/roxiemem/roxiemem.cpp

```diff
@@ -4157,46 +4173,58 @@ void * CChunkedHeap::inlineDoAllocate(unsigned allocatorId, unsigned maxSpillCos
     //The latter is done outside the spinblock, to reduce the window for contention.
     ChunkedHeaplet * donorHeaplet;
     char * chunk;
+    for (;;)
```

Comment:
It would be more consistent.  Probably because I've been wondering if we should use for(;;) instead of loop to make it make conventional.  I'll change.